src/Dapr.Actors.AspNetCore/ActorsEndpointRouteBuilderExtensions.cs

```diff
@@ -95,19 +95,7 @@ private static IEndpointConventionBuilder MapActorMethodEndpoint(this IEndpointR
                 }
                 else
                 {
-                    // write exception info in response.
-                    try
-                    {
-                        await runtime.DispatchWithoutRemotingAsync(actorTypeName, actorId, methodName, context.Request.Body, context.Response.Body);
-                    }
-                    catch (Exception e)
-                    {
-                        context.Response.Headers.Add("Connection: close", default(string));
```

Comment:
This case was very not likely having the desired effect anyway. This isn't a valid header 😆